src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -569,124 +536,107 @@ export class ServerView extends DirMixin(PolymerElement) {
       <paper-button on-tap="closeDataLimitsHelpBubble">[[localize('ok')]]</paper-button>
     </outline-help-bubble>
     `;
-    }
-
-    static get is() {
-      return 'outline-server-view';
-    }
-
-    static get properties() {
-      return {
-        serverId: String,
-        serverName: String,
-        serverHostname: String,
-        serverVersion: String,
-        isHostnameEditable: {type: Boolean},
-        serverManagementApiUrl: String,
-        serverPortForNewAccessKeys: Number,
-        isAccessKeyPortEditable: {type: Boolean},
-        serverCreationDate: String,
-        serverLocation: String,
-        accessKeyDataLimit: {type: Object},
-        isAccessKeyDataLimitEnabled: {type: Boolean},
-        supportsAccessKeyDataLimit: {type: Boolean},
-        dataLimitsAvailabilityDate: {type: String},
-        isServerManaged: Boolean,
-        isServerReachable: Boolean,
-        retryDisplayingServer: Function,
-        myConnection: Object,
-        totalInboundBytes: Number,
-        accessKeyRows: {type: Array},
-        hasNonAdminAccessKeys: Boolean,
-        metricsEnabled: Boolean,
-        monthlyOutboundTransferBytes: {type: Number},
-        monthlyCost: {type: Number},
-        selectedTab: {type: String},
-        managedServerUtilzationPercentage: {
-          type: Number,
-          computed:
-              '_computeManagedServerUtilzationPercentage(totalInboundBytes, monthlyOutboundTransferBytes)',
-        },
-        accessKeySortBy: {type: String},
-        accessKeySortDirection: {type: Number},
-        localize: {type: Function, readonly: true},
-      };
-    }
-
-    static get observers() {
-      return [
-        '_accessKeysAddedOrRemoved(accessKeyRows.splices)',
-        '_myConnectionChanged(myConnection)',
-        '_selectedTabChanged(selectedTab)',
-      ];
-    }
```

Comment:
@alalamav I'm not sure if this is still needed because I didn't see any code using these observers. I removed these and things seem fine, but I want to double check with you.

Reply:
I believe we still need it to hide the bubbles. You can test this behavior by deleting `localStorage['dataLimitsHelpBubble-dismissed'|'"addAccessKeyHelpBubble-dismissed"']`

Reply:
I'm pretty sure they are needed. You won't find any callers because they are called by the Polymer framework when the declared parameter changes.

Since you are getting rid of `myConnection`, `_myConnectionChanged` is probably safe to remove. But you should check the behavior of the other two. I think they dismiss the bubbles.

Reply:
Also: this observers feature isn't that great, since it doesn't allow for static analysis :-(
More on Polymer observers: https://polymer-library.polymer-project.org/3.0/docs/devguide/observers

Reply:
For the selectedTab observer, move it to the property declaration:
```js
        selectedTab: {type: String, observer: '_selectedTabChanged'},	
```

Reply:
You're both right. We still need _selectedTabChanged to dismiss help bubbles when we switch tabs.

But I think the other two can be removed because they both deal with special casing the admin key.